lib/genome/online_updater.rb

```diff
@@ -24,6 +24,11 @@ def create_gene_claim_attribute(gene_claim, name, value)
       ).first_or_create
     end
 
+    def create_gene_claim_category(gene_claim, category)
+      gene_category = DataModel::GeneClaimCategory.find_by(name: category)
```

Comment:
Unsure if this is correct or if it should be something like `gene_category = DataModel:: GeneClaimCategory.where(
        type: type.strip()
      ).first_or_create`

Reply:
I think this is fine if we think that our current set of categories is all that we want. Any new ones we decide on would need to be added manually. This would also prevent us from accidentally importing junk categories. You'll probably want to throw some sort of error here if gene_category is nil.